integration-tests/src/test/java/org/wildfly/prospero/it/cli/InstallTest.java

```diff
@@ -55,4 +55,28 @@ public void testInstallWithProvisionConfig() throws Exception {
                 .execute()
                 .assertReturnCode(ReturnCodes.SUCCESS);
     }
+
+    @Test
+    public void testInstallWithLocalRepositories() throws Exception {
+        final Path manifestPath = temp.newFile().toPath();
+        final Path provisionConfig = temp.newFile().toPath();
+        MetadataTestUtils.copyManifest("manifests/wfcore-base.yaml", manifestPath);
+        MetadataTestUtils.prepareChannel(provisionConfig, List.of(manifestPath.toUri().toURL()));
+
+        install(provisionConfig, targetDir.toPath());
+
+        upgradeStreamInManifest(manifestPath, resolvedUpgradeArtifact);
+
+        final URL temporaryRepo = mockTemporaryRepo(true);
+
+        Path relativePath = Paths.get(System.getProperty("user.dir")).relativize(Path.of(Path.of(temporaryRepo.toURI()).toUri()));
+
+        ExecutionUtils.prosperoExecution(CliConstants.Commands.UPDATE, CliConstants.Commands.PERFORM,
+                        CliConstants.REPOSITORIES, ".."+File.separator+"integration-tests"+File.separator+relativePath,
```

Comment:
You can use Path.of(String...) to build the path instead of appending the strings.

Also, `relativePath` is already a relative path, so I don't think adding `../integration-tests" to it brings anything to the test. I'd keep it simple and just use the `relativePath` here

Reply:
`target/folder1/folder2` is a relative path. An absolute path starts with root directory (e.g. `/home/jboss/` on linux or `C:\Users\jboss` on Windows). A relative path doesn't have a root folder and is relative to the current folder (e.g. `target/foo` is relative because it depends on where the user is)